### ISSUE TITLE
[Bug] Facts before Birth] 

### ISSUE THREAD

**Issue by buzzarddavis** [ShammyLevva/FTAnalyzer#255]
**Version Number problem appears in** 8.5.1.0.

**Describe the bug**Told that two people (married) have Facts Dated Before Birth. Both have the date of birth, the date of their marriage, occupation and residing (the latter two have abt dates). All are at or after their births

**To Reproduce** Not sure what you want. I used the FTAnalyzer, the printout of errors listed the problem and I do not know why

**Expected behavior** Describe problem or remove the error label for those people.

**Screenshots**
![image](https://user-images.githubusercontent.com/98553556/151430975-bbe334ba-deff-4f1e-b2db-575dbca7cb7a.png)
.

**Comment by ShammyLevva** [comment]
It looks like the fact details are mysteriously omitted from the description field. Not seen that one before. 

Can you export just those two individuals from your file to a small GEDCOM snippet. Load the snippet into your version to check the problem still occurs. Then email me the file to ftanalyzer@gmail.com I’ll then be able to have a look and see why the error is happening. 

Thanks for the report.

**Comment by buzzarddavis** [comment]
This is a quick note to say that I am not ignoring you.  I started to respond to you saying that I had started to follow your directions.  But Friday, I received two emails with information on family members and pictures.  So that took precedence I am afraid.  I plan on working on the Facts Before Birth problem today.
Howland Davis

**Comment by buzzarddavis** [comment]
Alexander:
Thank you for responding to my problem.  I am learning as I go here as far as exporting individuals.  1. I brought the two people up on the monitor2. I I clicked on 'File', 'Export' and 'GEDCOM'.3. I had to try several times to get what you wanted.  I have pasted the items that are in the download dealing with the whole family and then the Zitz couple in case I need to attach something else.4. I think that I have attached what you want. The FTAnalyzer shows 'Facts dated before birth' still.
Howland Davis



-----Original Message-----
